sycl/doc/XilinxFPGACompilation.md

```diff
@@ -31,6 +31,36 @@ Installing Xilinx FPGA compatible software stack:
   2. Xilinx runtime (XRT) for FPGAs: Download, build and install [XRT](https://github.com/Xilinx/XRT), this contains the OpenCL runtime.
   3. Xilinx SDx (2018.3+): Download and Install [SDx](https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/sdx-development-environments.html) which contains the `xocc` compiler.
 
+## Platforms
+
+It's of note that the SDx 2018.3 install comes with several platforms that do
+not work with the SYCL compiler, i.e. the ZYNC family of boards. Instead, you'll
+have to use one of the newer boards, like the Alveo U250 (*xilinx_u250_xdma_201830_1*).
+This requires some additional installation steps for the moment as it doesn't come
```

Comment:
What do you mean by "for the moment"? Do you have some view about the future?

Reply:
Ah no, I thought I removed that before pushing. I realized the wording made it sound like the platforms will eventually be included in the SDx install, but who knows if they ever will be.